src/integration-test.ts

```diff
@@ -201,7 +201,7 @@ describe('Nativefier upgrade', () => {
 describe('Browser version retrieval', () => {
   test('get chrome version with electron version', async () => {
     await expect(getChromeVersionForElectronVersion('12.0.0')).resolves.toBe(
-      '89.0.4389.69',
+      '102.0.5005.167',
```

Comment:
Please remove this change. This is actually what gets returned when we get an error trying to fetch the electron json file (which our prior url was deprecated). In the latest master branch I've fixed this to a good url. I saw you attempted to do so as well, but since you only linked to the GitHub file (and not the raw GitHub file), it didn't seem to be working.